packages/@aws-cdk/aws-appmesh/lib/shared-interfaces.ts

```diff
@@ -167,6 +169,48 @@ class FileAccessLog extends AccessLog {
   }
 }
 
+/**
+ * Represents the properties needed to define TLS validation context
+ */
+export interface TlsValidationContext {
+  /**
+   * Reference to where to retrieve the trust chain.
+   */
+  readonly trust: TlsValidationContextTrust;
+}
```

Comment:
Well noted.

Reply:
Well noted.

Just to be clear, although I renamed the file to `tls-validation-context.ts`, I am retaining file content as it is except moving `TlsValidationContext` interface.

Reply:
Will `subjectiveAlternativeNames` need to be interface or enum? Since in the CloudFormation, `subjectiveAlternativeNames.match.exact` is string type, I am thinking it might be better to create `TlsValidationContext` abstract instead and its factory methods assign both `subjectiveAltenativeNames` and `validation` for better prop flattening.

Reply:
Spoke with AJ, potentially `subjectievAlternativeNames.match` might be suitable for flattening but not all the way to `.exact` since we could be adding more options (e.g: `prefix` or `suffix`)